synapse/logging/opentracing.py

```diff
@@ -259,6 +259,14 @@ def report_span(self, span):
 logger = logging.getLogger(__name__)
 
 
+class SynapseTags:
+    # The message ID of any to_device message processed
+    TO_DEVICE_MESSAGE_ID = "to_device.message_id"
+
+    # Whether the sync response has new data to be returned to the client.
+    SYNC_RESULT = "sync.new_data"
```

Comment:
Is the purpose of this to check whether a `to_device` message came down `/sync` for a user? If so, then won't events other than `to_device` messages lower the signal-to-noise ratio?

Reply:
Ish, I was more interested in signalling whether we'd found anything or not. I might go in and add per-section flags later, but I'm a bit wary of adding toooo much at once